src/Node/BreakNode.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Node;
+
+use Twig\Compiler;
+
+final class BreakNode extends Node
+{
+    private int $level;
+
+    public function __construct(int $level)
+    {
+        parent::__construct([], [], 0, null);
```

Comment:
passing `0` as lineno is a no-go. the line of the node in the source must be given. Otherwise, it can do weird things for the error reporting.
And I would also pass a non-empty tag name.

Reply:
thank you, I've added the line number and tag name, however I'm not 100% sure about the two `arrays` here